tests/e2e/pkg/tester/skip_regex.go

```diff
@@ -62,8 +62,6 @@ func (t *Tester) setSkipRegexFlag() error {
 		skipRegex += "|TCP.CLOSE_WAIT"
 		// https://github.com/cilium/cilium/issues/15361
 		skipRegex += "|external.IP.is.not.assigned.to.a.node"
-		// https://github.com/cilium/cilium/issues/14287
-		skipRegex += "|same.port.number.but.different.protocols|same.hostPort.but.different.hostIP.and.protocol"
```

Comment:
Guess this is why GCE tests are failing:
```
Summarizing 2 Failures:
  [FAIL] [sig-network] HostPort [It] validates that there is no conflict between pods with same hostPort but different hostIP and protocol [LinuxOnly] [Conformance]
  [FAIL] [sig-network] Services [It] should be able to update service type to NodePort listening on same port number but different protocols

Reply:
oops, had that sitting in my local repo for cilium testing